tests/installation/welcome.pm

```diff
@@ -111,6 +111,20 @@ sub run {
         $self->accept_license;
     }
 
+    # Verify install arguments passed by bootloader
+    # Linuxrc writes its settings in /etc/install.inf
+    if (get_var('NETBOOT') && !check_var('BACKEND', 'ipmi')) {
+        wait_screen_change { send_key 'ctrl-alt-shift-x' };
+        my $method     = uc get_required_var('INSTALL_SOURCE');
+        my $mirror_src = get_var("MIRROR_$method");
+        die "Install source mismatch in boot parameters!\n"
```

Comment:
Maybe it's easier to read with `die` and `if` on the same line, no?

Reply:
Thanks for feedback! I have got the impression that the line was bit long. If you find it hard to read, I am happy to change it.

Reply:
I agree with the point that line become to long, but with two lines it's more confusing. To my taste I would use standard `if` syntax here with curly brackets.